app/src/main/res/layout/drawer_fragment.xml

```diff
@@ -66,7 +68,7 @@
             android:layout_height="24dp"
             android:layout_gravity="center_vertical"
             app:srcCompat="@drawable/ic_baseline_code_24"
-            android:tint="@{footerViewModel.isDeveloperOptionsSelected ? @color/highlighted_developer_options_nav_menu_item : @color/oppia_primary_text_dark}"/>
+            android:tint="@{footerViewModel.isDeveloperOptionsSelected ? @color/color_def_persian_blue : @color/component_color_drawer_fragment_developer_options_text_color}"/>
```

Comment:
Do not use color from `color_defs.xml` file. Use colors only from `component_colors.xml` file